.github/pull_request_template.md

```diff
@@ -1,6 +1,4 @@
-<!-- Important! Please follow the new guidelines for naming Pull Requests: https://docs.dasch.swiss/developers/dsp/contribution/#pull-request-guidelines-->
-
-Resolves DEV-
+<!-- Important! Please follow the guidelines for naming Pull Requests: https://docs.dasch.swiss/latest/developers/dsp/contribution/ -->
 
 ## PR Checklist
```

Comment:
This is IMO misleading, because `PR Type` is another checklist to fill.

How about making `PR checklist` main section and all other checklist add as subsections:
```
## PR Checklist

### PR Requirements

### PR Type

### Does this PR introduce a breaking change?

### Does this PR change client-test-data?
```

Reply:
Thanks for your feedback @mpro7 , I already merged it because my other PR is dependent on this one. I will add your suggestions in my open PR.